sdk/tables/azure-data-tables/tests/test_table_batch.py

```diff
@@ -1206,3 +1210,41 @@ def test_batch_url_for_cosmos_emulator(self):
         assert table.scheme == "http"
         with pytest.raises(RequestCorrect):
             table.submit_transaction(self.batch)
+
+    def test_decode_string_body(self):
+        def patch_run(request: PipelineRequest, **kwargs) -> PipelineResponse:
+            response = Response()
+            response.status_code = 405
+            response._content = b"<!DOCTYPE html><html><head><title>UnsupportedHttpVerb</title></head><body><h1>The resource doesn't support specified Http Verb.</h1><p><ul><li>HttpStatusCode: 405</li><li>ErrorCode: UnsupportedHttpVerb</li><li>RequestId : 98adf858-a01e-0071-2580-bfe811000000</li><li>TimeStamp : 2023-07-26T05:19:26.9825582Z</li></ul></p></body></html>"
```

Comment:
What request are we sending to get an "UnsupportedHttpVerb" error?
Do wehave a recording?